src/statmodels.jl

```diff
@@ -150,6 +150,14 @@ abstract RegressionModel <: StatisticalModel
 fitted(obj::RegressionModel) = error("fitted is not defined for $(typeof(obj)).")
 model_response(obj::RegressionModel) = error("model_response is not defined for $(typeof(obj)).")
 residuals(obj::RegressionModel) = error("residuals is not defined for $(typeof(obj)).")
+
+"""
+    predict(obj::RegressionModel, [newX])
+
+Form the predicted response of model `obj`. New covariate values `newX` can be supplied as a `DataFrame`
+with the same names as the original predictors, or as an AbstractMatrix (for low-level use) with a column
+for the value of each predictor (including a `Vector` of 1s if an intercept was fitted). 
+"""
```

Comment:
The point is, all these methods are going to be moved to StatsModels. So I don't think it's worth the trouble of adding partial docstrings everywhere just to remove them in a few weeks.